include/biscuit/assembler.hpp

```diff
@@ -1087,6 +1087,8 @@ class Assembler {
     void VRSUB(Vec vd, Vec vs2, GPR rs1, VecMask mask = VecMask::No) noexcept;
     void VRSUB(Vec vd, Vec vs2, int32_t simm, VecMask mask = VecMask::No) noexcept;
 
+    void VNEG(Vec vd, Vec vs, VecMask mask = VecMask::No) noexcept;
```

Comment:
Bit of a nitpick, but I'd move this declaration underneath the VNCLIPU instructions, so that the sequencing stays alphabetically organized